modules/@apostrophecms/page-type/index.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+  init(self) {
+    // convert old two-columns to columns
+    self.apos.migration.add('columns', async () => {
```

Comment:
This didn't work for me when I went from the content branch to content-columns. But I also don't think we need it. The project has a new shortname and you're the only one who worked on the content branch so far.

Reply:
how do you mean 'didn't work' ?

I guess the pathway to see something happen here would be:
- sync-down staging via the `main` branch (original shortname)
- switch to `content-columns` and run migration

These branches will eventually be merged into `main` and have their shortnames switched back, so I do think it needs to work ..

Reply:
I started the app on the content branch and added a two col widget to the home page. I then shut it off, switched branches, and tried to start up again. the widget disappeared rather than updating.